src/Frontend/package.json

```diff
@@ -11,9 +11,9 @@
     "type-check": "vue-tsc --build --force",
     "lint": "eslint .",
     "preview": "vite preview",
-    "test:component": "vitest ./src",
+    "test:component": "vitest run ./src",
```

Comment:
By default vitest runs in watch mode, but in our CI we always want it to perform a single run without watch mode.